### PR TITLE
[CDAP-16909] Fixes showing error message while saving/running pipeline with invalid profile

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA - https://cdap.atlassian.net/browse/CDAP-16909

### CODE REVIEW

File: cdap-ui/cypress/integration/pipeline.runtimeargs.spec.ts [hunk 1]
```diff
@@ -289,6 +289,17 @@ describe('Deploying pipeline with temporary runtime arguments', () => {
       )}`
     ).should('not.exist');
   });
+
+  it('should fail running with invalid runtime arguments', () => {
```
Comment (njbriggs) [inline cdap-ui/cypress/integration/pipeline.runtimeargs.spec.ts:293]: Should the test name specifically reference `profile`? Is profile handled differently from other runtime args?

**Review by njbriggs — APPROVED** [review]
LGTM, just 1 question about test names

File: cdap-ui/cypress/integration/pipeline.runtimeargs.spec.ts (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/cypress/integration/pipeline.runtimeargs.spec.ts:293]: The profile can be a runtime argument. The reference to profile as runtime argument is just an example in the JIRA. It doesn't have to be profile.

**Pull request merged**